### ISSUE TITLE
dont send photo

### ISSUE THREAD

**Issue by pedram1992** [akalongman/php-telegram-bot#268]
```php
$data['chat_id'] = $query->getMessage()->getChat()->getId();
$path = realpath('/Calendar.png');
Request::sendPhoto($data, $path);
```

Hi, i used `Request::sendPhoto($data, $path)` for send photo from local ,but don't send photo to user.

**Comment by imidazil** [comment]
$telegram->setUploadPath(__DIR__ . "/Uploads");
	$data = [
	                 'chat_id' => $chat_id
	             ];
    $result = Request::sendPhoto($data, $telegram->getUploadPath() . "/image.jpg");

**Comment by noplanman** [comment]
Hi!

I think the path to the file is wrong. Are you sure the path to `Calendar.png` is correct? It's searching for it in the `/` root folder, not the current one.

Try this if the file is in the current folder:
```php
$path = realpath(__DIR__ . '/Calendar.png');
```

You can also [enable logging](https://github.com/akalongman/php-telegram-bot/blob/0.35/examples/hook.php#L44-L46) to help find the problem.

**Comment by pedram1992** [comment]
@imidazil i tried many paths but not solve my problem (__DIR__ or __FILE__). do you test SendPhoto to your project with local photo?

**Comment by jacklul** [comment]
Maybe you're not using newest library version?

**Comment by noplanman** [comment]
Before calling `Request::sendPhoto`, add this line to make sure that the path is valid.
```php
Longman\TelegramBot\TelegramLog::debug($path);
```
Then check your debug log file and check that the correct path is output.

As @jacklul mentions, what version of the library are you using?

Also, remember to `return` the result at the end of your commands `execute()` method.

I assume that all other commands work properly as they should?

**Comment by pedram1992** [comment]
@jacklul i got it in 17 jul 2016.
@noplanman i checked debug log, i faced with **Bad Request: there is no photo in the request**  error.
all of commands work properly.

**Comment by noplanman** [comment]
Ok, then the version will be the problem.

The photo sending was fixed in version 0.35, you probably still have 0.34.

When you run the `/help` command, it tells you which version you have.

**Comment by pedram1992** [comment]
thanks @noplanman , i should update my akalongman to .035
